### PR TITLE
Use ID_VENDOR_ID and ID_MODEL_ID fallbacks

### PR DESCRIPTION
**Pull request by hughsie**
Some udev subsystems run probers to populate the values, rather than
just the kernel providing the sysfs files. Support reading these too.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Review by superm1 — COMMENTED** [review]
Do we have to worry about those racing with the daemon now though?

**Comment by hughsie** [comment]
I don't think so; the probers are run sync before the udev message.

File: libfwupdplugin/fu-udev-device.c [hunk 1]
```diff
@@ -325,6 +325,21 @@ fu_udev_device_probe_serio(FuUdevDevice *self, GError **error)
 	return TRUE;
 }
 
+static guint16
+fu_udev_device_get_property_as_uint16(GUdevDevice *udev_device, const gchar *key)
+{
+	const gchar *tmp = g_udev_device_get_property(udev_device, key);
+	guint64 value = 0;
```
Comment (superm1) [inline libfwupdplugin/fu-udev-device.c:332]: if you're capping the stoull call at G_MAX_UINT16, this should probably be a guint16.

**Review by superm1 — APPROVED** [review]
one nit; otherwise LGTM

File: libfwupdplugin/fu-udev-device.c (hunk 1 above)
Comment (hughsie) [reply libfwupdplugin/fu-udev-device.c:332]: nah, we pass by ref, and casting a different uint type isn't always portable.

**Comment by hughsie** [comment]
> one nit; otherwise LGTM

I'll check this works with Logitech and if so will merge, thanks.

**Pull request merged**